lib/path.js

```diff
@@ -0,0 +1,84 @@
+'use strict';
+
+const {PATH_SEPARATOR} = require('./constants');
+const isArray = require('./is-array');
+const isSymbol = require('./is-symbol');
+
+module.exports = {
+	after: (path, subPath) => {
+		if (isArray(path)) {
+			return path.slice(subPath.length);
+		}
+
+		if (subPath === '') {
+			return path;
+		}
+
+		return path.slice(subPath.length + 1);
+	},
+	concat: (path, key) => {
+		if (isArray(path)) {
+			path = path.slice();
+
+			if (key) {
+				path.push(key);
+			}
+
+			return path;
+		}
+
+		if (key && key.toString !== undefined) {
+			if (path !== '') {
+				path += PATH_SEPARATOR;
+			}
+
+			if (isSymbol(key)) {
+				return path + 'Symbol(' + key.description + ')';
```

Comment:
```suggestion
				return path + `Symbol(${key.description})`;
```

Reply:
The benchmarks for this function drop over 10% when I make this change,  so I would rather keep it if that's alright with you.

Reply:
Up to you. I would argue that if the benchmarks are affected by this, the benchmarks are faulty and focusing too much on microbenchmarking and not on real-world scenarios. Also make sure you're testing in the latest Node.js version.